maventa/types/MaventaFile.ts

```diff
@@ -0,0 +1,9 @@
+interface MaventaFile {
+    id: string;
+    filename: string;
+    type: string;
+    mimetype: string;
+    href: string;
+}
```

Comment:
You should use immutable readonly data structures unless there is a reason to have writable. In that case, better rename to "WritetableMaventaFile" or something like that. 

Otherwise, add `readonly` to every property.